src/components/fragments/popups/SchedulePopup.tsx

```diff
@@ -6,19 +6,25 @@ import { Row } from "react-bootstrap";
 import Date from "../forms/api/Date";
 import Time from "../forms/api/Time";
 import Button from "../util/Button";
+import { useNavigate } from "react-router-dom";
 
 export interface ScheduleParams {
   onSave: (start: string, end: string) => void,
   date?: string,
+  info?: string,
 }
 
 const SchedulePopup = (props: Readonly<ScheduleParams>) => {
   const [start, setStart] = useState("08:00");
   const [end, setEnd] = useState("16:00");
   const [error, setError] = useState("");
+  const [url, setUrl] = useState(props.info);
```

Comment:
Schedule Popup's only function is time editing. Additional functionality should be implemented through onSave event.